src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -3,30 +3,77 @@
 import core.basesyntax.dao.CommentDao;
 import core.basesyntax.model.Comment;
 import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
 
 public class CommentDaoImpl extends AbstractDao implements CommentDao {
     public CommentDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
     @Override
-    public Comment create(Comment entity) {
-        return null;
+    public Comment create(Comment comment) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(comment);
```

Comment:
Maybe we need to use persist() here. I think this was mentioned in a Q&A last week.

Reply:
done